### ISSUE TITLE
Collapsed active pin tabs have different shape than normal tabs

### ISSUE THREAD

**Issue by nanocryk** [Uiniel/zen-mods#6]
Pinned tab:
![image](https://github.com/user-attachments/assets/bb66684e-735a-4309-971f-c116e4f45737)

Normal tab:
![image](https://github.com/user-attachments/assets/cad16f43-8f8a-4311-8b80-70fabbc9c1b7)

I actually find that the pinned tab shape looks better, so while making both kind of tabs the same shape, make add an setting to make both look like this different shape?

**Comment by Uiniel** [comment]
yeah the pinned tab shape is actually a bug, but I can probably add this as an option
